Changelog.md

```diff
@@ -1,5 +1,67 @@
 ## Gazebo Sim 7.x
 
+### Gazebo Sim 7.8.0 (2024-07-22)
+
+1. Added support for spacecraft thrusters
+    * [Pull request #2431](https://github.com/gazebosim/gz-sim/pull/2431)
+
+1. Disable rendering tests that are failing on github actions
+    * [Pull request #2480](https://github.com/gazebosim/gz-sim/pull/2480)
+
+1. Merge from ign-gazebo6
+    * [Pull request #2477](https://github.com/gazebosim/gz-sim/pull/2477)
```

Comment:
I think we usually skip merge forward PRs in changelog.

Reply:
I'll remove this in a subsequent commit.